### DISCUSSION TITLE
Encountering a permission issue when trying to push the changes.

### DISCUSSION THREAD

**Discussion by hbSrujana** [vishanurag/Canvas-Editor#664] [Q&A] [answered]
Hi @vishanurag,
I’ve implemented the fix for the like button functionality and created a new branch `fix-like-button-issue`. However, I’m encountering a permission issue when trying to push the changes. 

 it shows an error like this :
$ git push origin fix-like-button-issue
info: please complete authentication in your browser...
remote: Permission to vishanurag/Canvas-Editor.git denied to hbSrujana.
fatal: unable to access 'https://github.com/vishanurag/Canvas-Editor.git/': The requested URL returned error: 403

Best regards,  
Srujana

**Comment by vishanurag** [comment, marked as answer]
> Hi @vishanurag, I’ve implemented the fix for the like button functionality and created a new branch `fix-like-button-issue`. However, I’m encountering a permission issue when trying to push the changes.
> 
> it shows an error like this : $ git push origin fix-like-button-issue info: please complete authentication in your browser... remote: Permission to vishanurag/Canvas-Editor.git denied to hbSrujana. fatal: unable to access 'https://github.com/vishanurag/Canvas-Editor.git/': The requested URL returned error: 403
> 
> Best regards, Srujana

@hbSrujana You need to fork the repo first and then clone/pull the forked repo only, not the original one. Then you can make changes in your branch locally. After that you will be able to push your code. Then simply create a pr.

**Comment by hbSrujana** [comment]
Ok, I'll check it out.

**Comment by hbSrujana** [comment]
Subject : Merge Conflicts Resolved
I just wanted to let you Know that I've resolved all merge conflicts, which
addresses issue #596  and PR is #737. The changes have been successfully
merged into the main branch
